Answers.md

```diff
@@ -1,2 +1,32 @@
 # Your responses to the short answer questions should be laid out here using Mark Down.
 ### For help with markdown syntax [Go here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+
+#*Questions*
+1. Describe some of the differences between .forEach & .map.
+···Both iterate over arrays but they have different behaviours: .forEach makes use of the referenced function once for each element in order (left to right) in the array, but does not change the rray and doesnt return anything. With .map the whole array is mutated and passed into a new array, applying the provided function.
```

Comment:
I didn't actually realise that `forEach` returns `undefined`. Nice catch! I learn something (more like a million things) from you guys every single day.